composer.json

```diff
@@ -36,25 +36,25 @@
     "require": {
         "php": "^7.2",
         "ext-ctype": "*",
-        "doctrine/annotations": "~1.5",
+        "ext-pdo": "*",
+        "doctrine/annotations": "~1.7",
         "doctrine/cache": "~1.6",
         "doctrine/collections": "^1.4",
-        "doctrine/dbal": "dev-snapshot/develop/2018-11-26 as 3.x-dev",
+        "doctrine/dbal": "dev-master",
```

Comment:
using `3.0.x-dev` should help us prevent issues with branch dancing (not that it should happen again)